lms/templates/courseware/progress.html

```diff
@@ -118,7 +118,7 @@ <h2 class="title">${_("Congratulations, you qualified for a certificate!")}</h2>
                 %elif credit_course_requirements['eligibility_status'] == 'partial_eligible':
                     <span>${student.get_full_name()}, ${_("You have not yet met the requirements for credit.")}</span>
                 %endif
-                <button class="credit-help"><i class="fa fa-question"></i><span class="sr">Help regarding credit requirement</span></button><br>
+                <a href="${settings.CREDIT_HELP_LINK_URL}" class="credit-help"><i class="fa fa-question"></i><span class="sr">Help regarding course credit requirements</span></a><br>
                 <div class="requirement-container" data-eligible="${credit_course_requirements['eligibility_status']}">
```

Comment:
I guess you need to make some changes in css as well because link is not appearing properly.
![Uploading Screen Shot 2015-07-01 at 2.40.36 PM.png…]()